ui/src/pages/insightDetail/components/eventAggregator/index.tsx

```diff
@@ -350,6 +351,9 @@
     ? { width: '100%', height: 300 }
     : { width: 400, height: 600 }
```

Comment:
Similarly, add a null check for `diagnosisEndRef.current` to prevent potential runtime errors: `diagnosisEndRef.current?.getBoundingClientRect()?.top`.